bindings/experimental/distrdf/python/DistRDF/Backends/Dask/Backend.py

```diff
@@ -193,9 +204,151 @@ def dask_mapper(current_range):
 
         return final_results.compute()
 
+    def ProcessAndMergeLive(self,
+                            ranges: List[Any],
+                            mapper: Callable[[Ranges.DataRange,
+                                            Callable[[Union[Ranges.EmptySourceRange, Ranges.TreeRangePerc]],
+                                                    Base.TaskObjects],
+                                            Callable[[ROOT.RDF.RNode, int], List],
+                                            Callable],
+                                            Base.TaskResult],
+                            reducer: Callable[[Base.TaskResult, Base.TaskResult], Base.TaskResult],
+                            drawables_info_dict: Dict[int, Tuple[Optional[List[Callable]], int, str]],
```

Comment:
Is it really a `Tuple[Optional[List[Callable]]` ? e.g. instances of this class could be
```python
t1 = (None, )
t2 = ([f1, f2, f3], )
```

Reply:
All clear now I got confused by the number of square brackets